addons/mass_mailing/views/utm_campaign_views.xml

```diff
@@ -84,7 +84,7 @@
                 <a name="%(action_view_mass_mailings_from_campaign)d" type="action"
                     t-attf-class="oe_mailings #{record.mailing_mail_ids.raw_value.length === 0 ? 'text-muted' : ''}"
                     groups="mass_mailing.group_mass_mailing_campaign">
-                    <t t-raw="record.mailing_mail_ids.raw_value.length"/> Mailings
+                    <t t-out="record.mailing_mail_ids.raw_value.length"/> Mailings
```

Comment:
I'm a bit concerned about using t-out in kanban templates, before having the t-out directive implemented in owl. We can't convert the kanban view to owl anymore, until this directive is supported (or we have to convert back t-out into t-raw, which would not be great).